src/rules/no-empty-source/README.md

```diff
@@ -0,0 +1,35 @@
+# no-empty-source
+
+Disallow empty source.
+
+```css
+  ···\n\t
+/**     ↑
+ *  This whitespace */
```

Comment:
I don't know ... all I can think of is `This empty source` or else no example. Either way.